### PR TITLE
Ticket chain definition + use

### PR DESCRIPTION
**Pull request by sternhenri**
Per our conversation last week @dignifiedquire @whyrusleeping @nicola .

The goal is for this definition to be succinct and clear as to how tickets are sampled. And to use this to clarify intended rule for sampling losing tickets for randomness in an edge case where these losing tickets yield a `TipSet` of multiple blocks.

### CODE REVIEW

File: definitions.md [hunk 1]
```diff
@@ -251,9 +251,13 @@ In the context of:
 
 A `ticket` is used as a source of randomness in EC leader election. Every block depends on an `ElectionProof` derived from a `ticket`. At least one new `ticket` is produced with every new block. Ticket creation is described [here](./expected-consensus.md#Ticket-generation).
 
+#### Ticket Chain
+
+Each blockchain in Filecoin can be associated to a given `ticket chain`. The `ticket chain` is assembled by taking the tickets (usually one) contained by the block with the smallest final ticket in each of the chain's `TipSet`s.
```
Comment (dignifiedquire) [inline definitions.md]: each block or each block chain?

File: expected-consensus.md [hunk 2]
```diff
@@ -195,8 +197,8 @@ Now, a miner must also check whether they have won in this round.
 
 The process is as follows:
 
-- At round N, the miner will draw the smallest ticket from the TipSet at round `N-K`, with `K` the randomness lookback parameter.
-  - In the case where there are multiple tickets to choose from at round `N-K` (i.e. if the TipSet eventually created has multiple blocks), miners should attempt to generate their `ElectionProof` from the ticket generated by the block with the smallest final ticket (i.e. not necessarily the smallest ticket generated at that round).
+- At round N, the miner will sample the `ticket chain` at round `N-K`, with `K` the randomness lookback parameter.
+  - This would typically mean the smallest ticket in the `TipSet` generated in that round, but in the case that round is did not yield any successful ElectionProofs (i.e. only generated losing tickets), miners should attempt to generate their new `ElectionProof` from the ticket generated by the eventual block generated with the smallest final ticket (i.e. not necessarily the smallest ticket generated at `N-K` -- this helps ensure liveness in case there are more than K successive losing tickets).
```
Comment (dignifiedquire) [inline expected-consensus.md]: typo: is did

File: definitions.md (hunk 1 above)
Comment (sternhenri) [reply definitions.md]: each chain I guess, so blockchain but that's an overloaded term

File: expected-consensus.md [hunk 3]
```diff
@@ -222,7 +225,14 @@ Succinctly, the process of crafting a new `ElectionProof` in round `N` is as fol
   - Sha256 for our hashing function
   - Secp256k1 for our curve
 
+<<<<<<< HEAD
```
Comment (dignifiedquire) [inline expected-consensus.md]: merge conflict detected

**Pull request merged**